README.md

```diff
@@ -171,6 +179,22 @@ Success!
 Your keys can be found at: <YOUR_FOLDER_PATH>
 ```
 
+###### `generate-bls-to-execution-change` Arguments 
+
+You can use `bls-to-execution-change --help` to see all arguments. Note that if there are missing arguments that the CLI needs, it will ask you for them.
+
+| Argument | Type | Description |
+| -------- | -------- | -------- |
+| `--bls_to_execution_changes_folder` | String. Pointing to `./bls_to_execution_changes` by default | The folder path for the `bls_to_execution_change-*` JSON file(s) |
+| `--chain` | String. `mainnet` by default | The chain setting for the signing domain. |
```

Comment:
This needs to be adopted with the lukso default chain

Reply:
Checking the diff.

Reply:
Added modifications to the --chain table row by copying it from another sub-command.